internal/pgbackrest/config.md

```diff
@@ -25,7 +25,7 @@ During initial cluster creation, four pgBackRest use cases are involved.
 These settings are configured in either the [global] or [stanza] sections of the 
 pgBackRest configuration based on their designation in the pgBackRest code.
 For more information on the above, and other settings, please see
-https://github.com/pgbackrest/pgbackrest/blob/master/src/config/parse.auto.c
+https://github.com/pgbackrest/pgbackrest/blob/release/2.38/src/config/parse.auto.c
```

Comment:
As of 5.1.3 and 5.2.0, we moved to pgbackrest 2.40, right? They changed some things and this page was replaced by `parse.auto.c.inc`, but which do we want to point to here?

Reply:
We have a few other instances and they all specifically point to the 2.38 page.

Reply:
Yup, agreed.  This should point to the current pgBackRest version we're using for v5 (which should always be the same for all v5 lines).  Good catch!  And something we need to ensure we update when we prep our releases moving forward (assuming the pgBackRest version changes)...

Reply:
This file contains implementation details, and the links are to the code we based decisions on. I'm not worried about these being up-to-date for users.